app/models/chess_piece.rb

```diff
@@ -84,4 +84,14 @@ def is_obstructed?(endpoint_x, endpoint_y)
     end
   end
 
+  def capture(endpoint_x, endpoint_y)
+    opponent_piece = game.piece_in_square?(endpoint_x, endpoint_y)
```

Comment:
this line will return a true or false. Maybe you're looking for something like this -
```
current_piece = chess_piece(position_x, position_y)
opponent_piece = chess_piece(endpoint_x, endpoint_y)
```